personalsite/src/App.js

```diff
@@ -0,0 +1,54 @@
+import logo from './logo.svg';
+import './App.css';
+import React from 'react';
+
+function App() {
+
+  const name = "Emily"
+  var nameComponent = (name== "Emily") ?
+  <p>hi my name is {name}</p> : <p>my  name is not Emily</p>;
+  const buttonFunction = () => {
+    console.log("the button was clicked");
+  }
+ 
+
+  return(
+    
+    <div className='App'>
+
+    <header className="App-header">
+        <div>{nameComponent}</div>
+       <button onClick={buttonFunction}>I am a button</button>
+     </header>
+   </div>
+      </header>
```

Comment:
this extra closing tag was the main thing stopping it from compiling!
every opening tag can only have 1 closing tag. the computer doesnt know how to read this because it doesnt have a `<header>` opening tag to match it with. so it errors out here and won't display anything